pypit/scripts/coadd_1dspec.py

```diff
@@ -41,7 +41,7 @@ def main(args, unit_test=False):
     files = []
     for ifl in filelist:
         if '*' in ifl:
-            files.append(glob.glob(ifl))
```

Comment:
Good catch!

Have edited it back.  Can't remember why this 
was causing me trouble but will solve it when I do.